2019/5xxx/CVE-2019-5631.json

```diff
@@ -1,18 +1,112 @@
 {
-    "CVE_data_meta": {
-        "ASSIGNER": "cve@mitre.org",
-        "ID": "CVE-2019-5631",
-        "STATE": "RESERVED"
-    },
-    "data_format": "MITRE",
-    "data_type": "CVE",
-    "data_version": "4.0",
-    "description": {
-        "description_data": [
-            {
-                "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
-            }
+  "data_type": "CVE",
+  "data_format": "MITRE",
+  "data_version": "4.0",
+  "generator": {
+    "engine": "Vulnogram 0.0.7"
+  },
+  "CVE_data_meta": {
+    "ID": "CVE-2019-5631",
+    "ASSIGNER": "cve@rapid7.com",
+    "DATE_PUBLIC": "2019-07-10T21:30:00.000Z",
+    "TITLE": "Rapid7 InsightAppSec Local Privilege Escalation",
+    "AKA": "",
+    "STATE": "PUBLIC"
+  },
+  "source": {
+    "defect": [],
+    "advisory": "R7-2019-26",
+    "discovery": "EXTERNAL"
+  },
+  "affects": {
+    "vendor": {
+      "vendor_data": [
+        {
+          "vendor_name": "Rapid7",
+          "product": {
+            "product_data": [
+              {
+                "product_name": "InsightAppSec",
+                "version": {
+                  "version_data": [
+                    {
+                      "version_name": "2019.06.24",
+                      "version_affected": "<=",
+                      "version_value": "2019.06.24",
+                      "platform": ""
+                    }
+                  ]
+                }
+              }
+            ]
+          }
+        }
+      ]
+    }
+  },
+  "problemtype": {
+    "problemtype_data": [
+      {
+        "description": [
+          {
+            "lang": "eng",
+            "value": "CWE-427: Uncontrolled Search Path Element"
+          }
         ]
+      }
+    ]
+  },
+  "description": {
+    "description_data": [
+      {
+        "lang": "eng",
+        "value": "Rapid7 InsightAgent suffers from a DLL injection vulnerability in the 'prunsrv.exe' component of the product. If exploited, a local user of the system (who must already be authenticated to the operating system) can elevate their privileges with this vulnerability to the privilege level of InsightAppSec (usually, SYSTEM). This issue affects version 2019.06.24 and prior versions of the product."
```

Comment:
should this be InsightAgent? maybe InsightAppSec broker instead?

Reply:
Oh sure thanks @lgreene-r7